Towny/src/main/java/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -705,10 +712,10 @@ private static void setDefaults(String version, Path configPath) {
 			} else if (root.getRoot().equals(ConfigNodes.LAST_RUN_VERSION.getRoot())) {
 				setNewProperty(root.getRoot(), getLastRunVersion(version));
 			} else if (root.getRoot().equals(ConfigNodes.TOWNBLOCKTYPES_TYPES.getRoot())) {
-				setNewProperty(root.getRoot(), root.getDefault());
+				setNewProperty(root.getRoot(), root.defaultValue());
```

Comment:
In my experience handling maps gets a bit weird, this is likely the point where townblocktypes stopped working. I ran into something similar in TownyChat when changing the channels.yml over to commented config.

Reply:
That's because setNewProperty used to always toString the value, but this is removed here